plugin-api/src/main/java/net/clgd/ccemux/api/emulation/EmulatedComputer.java

```diff
@@ -201,6 +201,17 @@ public void scroll(int lines, int x, int y) {
 		if (inTerminal(x, y)) queueEvent("mouse_scroll", new Object[] { lines, x, y });
 	}
 
+	/**
+	 * Queues a mouse move event
+	 */
+	public void move(int x, int y) {
+		if (inTerminal(x, y)) {
+			queueEvent("mouse_move", new Object[] { x, y });
```

Comment:
Where's the button arg?

Reply:
Mouse_move doesn’t need a button arg?

Reply:
I added the button argument, it's always `1`.